temba/orgs/models.py

```diff
@@ -86,6 +86,9 @@
 ACCOUNT_SID = 'ACCOUNT_SID'
 ACCOUNT_TOKEN = 'ACCOUNT_TOKEN'
 
+TWILIO_COMPLIANT_API_ACCOUNT_SID = 'TWILIO_COMPLIANT_API_ACCOUNT_SID'
+TWILIO_COMPLIANT_API_ACCOUNT_TOKEN = 'TWILIO_COMPLIANT_API_ACCOUNT_TOKEN'
```

Comment:
Sure totally agree. I'll have a look how the Kannel endpoint is setup and try to adapt this channel from there